src/contrib/sort/sort.h

```diff
@@ -0,0 +1,33 @@
+/*!
+ *  Copyright (c) 2017 by Contributors
+ * \file Use external generic C library function
+ */
+
+#ifndef TVM_CONTRIB_SORT_SORT_H_
+#define TVM_CONTRIB_SORT_SORT_H_
+
+#include <tvm/runtime/registry.h>
+
+namespace tvm {
+namespace contrib {
+
+template<typename DType>
+struct SortElem {
+  DType value;
```

Comment:
since this is only used by the sort.cc, not necessary to have a header, put it directly in the cc file